cmd/world/evm/evm.go

```diff
@@ -0,0 +1,183 @@
+package evm
+
+import (
+	"bytes"
+	"fmt"
+	"os/exec"
+	"time"
+
+	"github.com/spf13/cobra"
+	"pkg.world.dev/world-cli/common/config"
+	"pkg.world.dev/world-cli/common/tea_cmd"
+)
+
+func EVMCmds() *cobra.Command {
+	evmRootCmd := &cobra.Command{
+		Use:   "evm",
+		Short: "EVM base shard commands.",
+		Long:  "Commands for provisioning the EVM Base Shard.",
+	}
+	evmRootCmd.AddGroup(&cobra.Group{
+		ID:    "EVM",
+		Title: "EVM Base Shard Commands",
+	})
+	evmRootCmd.AddCommand(
+		StartEVM(),
+		StartDA(),
+		StopAll(),
+	)
+	return evmRootCmd
+}
+
+const (
+	FlagDAAuthToken = "da-auth-token"
+	EnvDAAuthToken  = "DA_AUTH_TOKEN"
+)
+
+func services(s ...tea_cmd.DockerService) []tea_cmd.DockerService {
+	return s
+}
+
+func StartDA() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "start-da",
```

Comment:
The DA layer docker service here is only used for local testing, in production it would just be a public HTTP endpoint from Celestia itself, so i would turn this into either `world evm dev` or `world evm start --dev` that would start the DA layer devnet for local testing + the EVM (there is no instance where we need to run DA without EVM)

The default world.toml should not contain default DA_AUTH_TOKEN, DA_BASE_URL, and DA_NAMESPACE_ID; just empty string is fine.

`world start` (for production) should check that users have set these values, but `world evm dev`/`world evm start --dev` would just inject 
DA_AUTH_TOKEN=$(docker exec $(docker ps -q)  celestia bridge --node.store /home/celestia/bridge/ auth admin)
DA_BASE_URL="http://celestia-devnet:26658"
DA_NAMESPACE_ID="67480c4a88c4d12935d4"

Reply:
I added a --dev flag to run the container locally. "prod" mode checks the 3 flags above are set.